subprojects/publish/src/main/java/org/gradle/api/publish/internal/GradleModuleMetadataWriter.java

```diff
@@ -259,15 +259,15 @@ private void writeVariants(PublicationInternal publication, SoftwareComponent co
         }
     }
 
-    private void writeCreator(JsonWriter jsonWriter) throws IOException {
+    private void writeCreator(PublicationInternal publication, JsonWriter jsonWriter) throws IOException {
         jsonWriter.name("createdBy");
         jsonWriter.beginObject();
         jsonWriter.name("gradle");
         jsonWriter.beginObject();
         jsonWriter.name("version");
         jsonWriter.value(GradleVersion.current().getVersion());
         jsonWriter.name("buildId");
-        jsonWriter.value(buildInvocationScopeId.getId().asString());
```

Comment:
I think this field is now unused

Reply:
`buildInvocationScopeId` is still used below 👇